src/middleware/fhir/controller.utils.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable security/detect-object-injection */
```

Comment:
Copy and change this code from the node-fhir-server-core so it can support streaming